src/components/ui/CoralIconButton.js

```diff
@@ -0,0 +1,62 @@
+import React, { Component,  PropTypes } from 'react';
+import Radium from 'radium';
+
+/**
+ * Pass as icon the name of the Icon
+ * Find them here: https://design.google.com/icons
+ */
+
+  // TODO(bc): Add support for sizes and icon sizes
+
+@Radium
+class IconButton extends Component {
+  render() {
+    const { onClick, icon, style, disabled, size, ...rest } = this.props;
+    return (
+      <button
+        onClick={onClick}
+        disabled={disabled ? 'disabled' : ''}
+        style={[styles.base]}
+        { ...rest }
+      >
+        <i
+          className="material-icons"
+          style={[styles.icon.base]}
+        >
+          {icon}
+        </i>
+      </button>
+    )
+  }
+}
+
+const styles = {
+  background: 'white',
+  base: {
+    color: '#262626',
+    backgroundColor: 'white',
+    border: 'none',
+    borderRadius: '4px',
+    border: 0,
+    padding: '6px',
+    borderRadius: '4em',
+    cursor: 'pointer',
+    outline: 'none',
+    ':hover': {
+      backgroundColor: '#d8d8d8'
+    },
+    ':focus': {
+      backgroundColor: '#d8d8d8'
+    },
+    ':active': {
+      backgroundColor: 'white',
+    }
+  },
+  icon: {
+    base: {
+      fontSize: 18
+    }
+  }
+};
+
+export const CoralIconButton = Radium(IconButton);
```

Comment:
if you're using the decorator `@Radium` on line 11, you don't have to wrap the export here -- you'd be performing the same action twice. [ES7 Decorators](https://github.com/wycats/javascript-decorators)